--- a/feeds/packages/net/travelmate/Makefile
+++ b/feeds/packages/net/travelmate/Makefile
@@ -20,7 +20,7 @@ define Package/travelmate
 	SECTION:=net
 	CATEGORY:=Network
 	TITLE:=A wlan connection manager for travel router
-	DEPENDS:=+iwinfo +jshn +jsonfilter +curl +ca-bundle +rpcd +rpcd-mod-rpcsys
+	DEPENDS:=+iwinfo-ucode +jshn +jsonfilter +curl +ca-bundle +rpcd +rpcd-mod-rpcsys
 	PKGARCH:=all
 endef
 
